firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -61,7 +60,7 @@
   FirebaseInstallations(FirebaseApp firebaseApp) {
     this(
         DefaultClock.getInstance(),
-        new ThreadPoolExecutor(0, 1, 30L, TimeUnit.SECONDS, new SynchronousQueue<>()),
```

Comment:
weren't we going to stick with this constructor but use a linked list for the queue instead? that way the thread's allowed to die - otherwise it just sits there idle wasting memory

Reply:
Got it. Updated. Thanks.